Quintana_Jerrod/Assignments/flask_fundamentals/great_number_game/ngame.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, render_template, session, redirect, request
+import random
+app = Flask(__name__)
+app.secret_key='This'
+
+@app.route('/')
+def start():
+    if not ('num') in session:
+        session['num'] = random.randrange(0,101)
+    print session['num']
+    num = session['num']
+    if ('guess') in session:
+        if int(session['guess']) == int(session['num']):
```

Comment:
`session['num']` is set as `session['num'] = random.randrange(0,101)`. In other words, `session['num']` is, and will always be, an int. No need to cast it to an int when it's already one =)